cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-manager/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: curator-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manager-role
```

Comment:
I don't see a matching role named `manager-role` anywhere. Is this meant to be the `curator-report-manager` role?

Reply:
Yes. It should be.